mu/interface.py

```diff
@@ -562,6 +563,18 @@ def __init__(self, parent):
                        tool_text="Show help about Mu in a browser.")
         self.addAction(name="quit", tool_text="Quit Mu.")
 
+    def setCompactMode(self, toggle):
+        """
+        Option for a compact button bar to be used when the window is very small.
+
+        """
```

Comment:
To follow convention with the rest of the docstrings it would be better to remove the additional breakline before the end of the comment.